src/api/app/controllers/webui/webui_controller.rb

```diff
@@ -88,6 +88,8 @@ class MissingParameterError < RuntimeError; end
   end
 
   def set_project
+    # We've started to use project_name for new routes...
+    params[:project] = params[:project_name] if params[:project_name].present?
```

Comment:
Hakiri thinks this is an XSS. But I think this is a false positive. Please someone double check...